### PR TITLE
Correcting a error

### PR DESCRIPTION
**Pull request by RLordan**
Sorry I made an error in vitamin D section that is now corrected.

### Suggested reviewers (optional)

@rando2 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -475,7 +475,7 @@ In terms of other dietary supplements, vitamin D can modulate the adaptive and i
...
 In particular, vitamin D deficient patients are at risk of developing acute respiratory infections [@doi:10.1016/j.jsbmb.2012.11.017] and ARDS [@doi:10.1016/j.jsbmb.2012.11.017].
-1,25-dihydroxyvitamin D3 (25-hydroxyvitamin D) is the active form of vitamin D that is involved in adaptive and innate responses, however, due to a short half life of a few hours, it is measured by its longer lasting precursor 25-dihydroxyvitamin D3. The vitamin D receptor is expressed in various immune cells and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes [@doi:10.2310/JIM.0b013e31821b8755; @doi:10.1016/j.coph.2010.04.001].
+1,25-dihydroxyvitamin D3 is the more active form of vitamin D that is involved in adaptive and innate responses, however, due to a short half life of a few hours, it is measured by its longer lasting precursor 25-hydroxyvitamin D. The vitamin D receptor is expressed in various immune cells and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes [@doi:10.2310/JIM.0b013e31821b8755; @doi:10.1016/j.coph.2010.04.001].
```
Comment (rando2) [inline content/20.treatments.md:478]:
```suggestion
1,25-dihydroxyvitamin D3 is the more active form of vitamin D that is involved in adaptive and innate responses; however, due to a short half life of a few hours, it is measured by its longer lasting precursor, 25-hydroxyvitamin D. 
The vitamin D receptor is expressed in various immune cells, and vitamin D is an immunomodulator of antigen presenting cells, dendritic cells, macrophages, monocytes, and T- and B-lymphocytes [@doi:10.2310/JIM.0b013e31821b8755; @doi:10.1016/j.coph.2010.04.001].
```

**Review by rando2 — APPROVED** [review]
Looks good to me!

**Comment by RLordan** [comment]
> Looks good to me!

I was writing about vitamin D early and wasn’t sure if I had said the same thing on the paper haha Will get a colleague to double check later to be sure

**Comment by rando2** [comment]
@RLordan just keep me posted if you want me to merge it in! :)

**Comment by RLordan** [comment]
> @RLordan just keep me posted if you want me to merge it in! :)

Hey @rando2 , thanks a million, I think we are good to merge :)

**Pull request merged**